src/Model/Table/SentencesTable.php

```diff
@@ -377,12 +379,56 @@ public function findFilteredTranslations($query, $options) {
                 $indirectTranslations = Hash::sort($indirectTranslations, '{n}.lang', 'asc');
                 
                 $result['translations'] = [$directTranslations, $indirectTranslations];
+                $result['extraTranslationsCount'] = $this->getextraTranslationsCount($result);
+                $result['expandLabel'] = $this->getExpandLabel($result['extraTranslationsCount']);
+                if (CurrentUser::isMember()) {
+                    $result['menu'] = $this->getMenu($result);
```

Comment:
I think you shouldn’t call that `menu` because we are in the context of the model, not the view. How about `perms`?